src/seedcase_sprout/core/sprout_checks/check_column_data_types.py

```diff
@@ -0,0 +1,207 @@
+import json
+
+import polars as pl
+
+from seedcase_sprout.core.map_data_types import FRICTIONLESS_TO_POLARS
+from seedcase_sprout.core.properties import FieldType
+
+# https://datapackage.org/standard/table-schema/#boolean
+BOOLEAN_VALUES = {"false", "False", "FALSE", "0", "true", "True", "TRUE", "1"}
+
+
+def check_is_boolean(field_name: str) -> pl.Expr:
+    """Checks if the column contains only boolean values.
+
+    Failed values are marked with False.
+
+    Args:
+        field_name: The name of the column to check.
+
+    Returns:
+        A Polars expression for checking the column.
+    """
+    return pl.col(field_name).is_in(BOOLEAN_VALUES)
+
+
+def check_is_castable_type(field_name: str, field_type: FieldType) -> pl.Expr:
```

Comment:
```suggestion
def check_is_given_type(field_name: str, field_type: FieldType) -> pl.Expr:
```

castable isn't super clear what it means. Maybe "given"? But I don't love that either ... :thinking: Or just `check_is_field_type()`